update pgbouncer from 1.8.1 to 1.14.0

fzhang0111 opened the pull request:
@jhwillett @fabien-copper , I did upstream merge to my branch in order to have newer version of pgbouncer. do not know if other changes from upstream will cause troubles or not, can you help  have a look, not in rush for merging to our master branch. Thanks!

fzhang0111 committed:
update pgbouncer

fzhang0111 committed:
Merge branch 'master' into zf-pgbouncer

jhwillett reviewed (approved):
This looks fine to me.

Note that these changes will not "go live" in ALI until we update this line: https://github.com/ProsperWorks/ALI/blob/dev/Dockerfile.ali#L220.

So it is safe for you to merge this forward in this repo, then make an ALI PR where you test it on `ali-integration-?`, etc.

fzhang0111 committed:
for pass Selinum test purpose

fzhang0111 committed:
need the package  y
Merge branch 'zf-pgbouncer' of https://github.com/ProsperWorks/heroku-buildpack-pgbouncer into zf-pgbouncer

fzhang0111 reviewed (commented):
add .travis.yml

fzhang0111 commented:
trigger a rerun

fzhang0111 committed:
rm .travis.yml
